packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -15,14 +15,12 @@ import Div100vh from 'react-div-100vh';
 /* To bypass usage of component altogether, use is_bypassed */
 
 type TDiv100vhContainer = {
-    id?: string;
     height_offset: string;
     is_bypassed?: boolean;
     is_disabled?: boolean;
     max_height_offset?: string;
-    className: string;
     max_autoheight_offset?: string;
-};
+} & ComponentProps<'div'>;
```

Comment:
```suggestion
} & React.ComponentProps<'div'>;
```
according to our project docs/convention, all React features should be accessed via `React.` instead of being imported directly :)